contracts/StakingDistributor.sol

```diff
@@ -1,211 +1,200 @@
 // SPDX-License-Identifier: AGPL-3.0
-pragma solidity ^0.7.5;
-
-import "./libraries/SafeERC20.sol";
-import "./libraries/SafeMath.sol";
+pragma solidity ^0.8.10;
+pragma abicoder v2;
```

Comment:
Don't need abicoder v2 in solidity 0.8. It's the default (see https://docs.soliditylang.org/en/v0.8.13/080-breaking-changes.html#silent-changes-of-the-semantics)

Reply:
implemented